packages/core/src/sass/app/_common/components/onfido-container.scss

```diff
@@ -72,191 +72,688 @@
+                }
+            }
+        }
+    }
+    .proof-of-identity__inner-container {
+        @include desktop {
+            &--incl-image {
+                display: grid;
+                align-items: center;
+                column-gap: 1.5rem;
+                grid-template-columns: auto 0.5fr;
+            }
+        }
+    }
+}
 
-        .account-form__fieldset {
-            max-width: unset;
+.idv-resubmit {
+    .poi-form-on-signup__fields {
+        .proof-of-identity__fieldset-container {
+            display: flex;
+            flex-direction: column;
+            gap: 3.2rem;
         }
+    }
+}
 
-        &--status-message {
-            margin-top: 1.6rem;
-            text-align: center;
-            width: 100%;
+.upload-layout {
+    height: 100%;
+    overflow: auto;
+    .account-form__poi-confirm-example {
+        margin: unset !important;
+        margin-bottom: 3.2rem !important;
+    }
+}
+
+.account {
+    margin: 0 16px;
+    &__scrollbars_container {
+        height: 100%;
+        padding-top: 2.4rem;
+        padding-bottom: 6.4rem;
+
+        &-wrapper {
+            overflow-x: hidden;
+            overflow-y: auto;
         }
-        &_container {
-            border: 1px solid var(--general-active);
-            border-radius: 0.8rem;
 
-            @include desktop() {
-                padding: 1.6rem;
+        &--grid-layout {
+            display: grid;
+            grid-gap: 4px;
+
+            .dc-input {
+                margin-bottom: 0;
             }
 
-            @include mobile() {
-                margin-bottom: 7rem;
-                padding: 1.6rem 1.6rem 0;
+            .dc-dropdown-container {
+                .dc-dropdown__display-text,
+                .dc-list__item-text {
+                    text-transform: unset;
+                }
             }
 
-            .account__scrollbars_container {
-                padding: 0;
+            & .account-form__header.address {
+                margin-top: 3rem;
+            }
 
-                &--grid-layout {
-                    height: auto !important;
+            @include mobile {
+                padding: 0 1.6rem;
+                overflow-x: hidden;
+                overflow-y: auto;
+                grid-template-rows: auto;
+
+                & .account-form__header {
+                    &:first-child {
+                        padding-top: 2.4rem;
+                        margin-bottom: 3.2rem;
+                    }
 
-                    @include mobile() {
-                        padding: 0 1.6rem;
+                    &.address {
+                        margin-top: 3rem;
                     }
                 }
+            }
+        }
+
+        @include desktop {
+            padding-left: 16px;
+            padding-right: 16px;
+        }
 
-                .account-form__section {
+        @include mobile {
+            height: unset;
+            padding-top: unset;
+            padding-bottom: unset;
```

Comment:
we can combine this right 
padding: unset;

Reply:
Only the top & bottom is to be removed. The left & right padding are as expected